man/osg-incommon-cert-request.1

```diff
@@ -44,41 +46,63 @@ May be specified more than once for additional SANs.
 .TP
 .BR \-d, \-\-directory
 The directory to write the host certificate(s) and key(s).
+.TP
+.BR \-O, \-\-orgcode
+Specify alternative organization and department codes for the InCommon \\
+Certificate Service. Defaults are Fermilab\'s codes.
 .SH EXAMPLES
 .PP
 To generate a certificate with no Subject Alternative Names (SANS)
 .PP
 .RS
 .nf
-osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem -H hostname.yourdomain
+osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem \\
+-H hostname.yourdomain
 .fi
 .RE
 .PP
 To generate a certificate with multiple Subject Alternative Names (SANS)
 .PP
 .RS
 .nf
-osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem -H hostname.yourdomain \\ 
--a hostalternative.yourdomain -a hostalternative2.yourdomain 
+osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem \\
+-H hostname.yourdomain -a hostalt.yourdomain -a hostalt2.yourdomain 
 .fi
 .RE
 .PP
 To generate multiple certificates from a hostfile
 .PP
 .RS
 .nf
-osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem -F hostfilepath.txt
+osg-incommon-cert-request -u incommonuser -c certpath.pem -k keypath.pem \\
+-F hostfilepath.txt
 .RE
 .PP
 hostfilepath.txt example 
 .PP
 .RS
 hostname01.yourdomain
-hostname02.yourdomain hostnamealias.yourdomain hostname03.yourdomain
+hostname02.yourdomain althostname.yourdomain hostname03.yourdomain
 hostname04.yourdomain hostname05.yourdomain
 .RE
 .fi
 .RE
+.PP
+To provide alternative organization and department codes for the InCommon \\
+Certificate Service. Codes can be found at the InCommon Cert Manager web \\
+interface (https://cert-manager.com/customer/InCommon): \\
+.IP \[bu] 2
+Organization Code is shown as OrgID under Settings > Organizations > Edit
+.IP \[bu]
+Department Code is shown as OrgID under Settings > Organizations > \\
+Departments > Edit
+.PP
+.RS
+.nf
+osg-incommon-cert-request -O 4567,8912 -u incommonuser -c certpath.pem \\
```

Comment:
shorten, and indent subsequent lines

Reply:
Fixed

Reply:
still needs indent

Reply:
Please check if the indents are correct.